monitor/census.go

```diff
@@ -251,6 +257,8 @@ func InitCensus(nodeType NodeType, version string) {
 	census.kOrchestratorAddress = tag.MustNewKey("orchestrator_address")
 	census.kFVErrorType = tag.MustNewKey("fverror_type")
 	census.kSegClassName = tag.MustNewKey("seg_class_name")
+	census.kRound = tag.MustNewKey("round")
```

Comment:
I'm not sure we want to have `round` and `tx_hash` as tags, because these are always unique values. So, from my understanding, each error will result in a new line from the `/metrics` endpoint. That makes me think that these tags are a better fit for logging, not Prometheus metrics. So, I'd suggest removing these tags from here.

To give a more specific example, let's say that calling reward failed 3 times. Here's what you'll see in the `/metrics` endpoint.

```
livepeer_reward_call_errors{error_code="RewardCallError",node_id="Rafas-MacBook-Pro.local",node_type="orch",round="1",tx_hash="0x3cf6917fc096db70a9ba2bd942db6746cfaeffafdbe510773bc481b51038d056"} 1
livepeer_reward_call_errors{error_code="RewardCallError",node_id="Rafas-MacBook-Pro.local",node_type="orch",round="2",tx_hash="0x316b0a9e17749f78b946f940cfc14403a3366b33d9bae99a9476756abe8b3f16"} 1
livepeer_reward_call_errors{error_code="RewardCallError",node_id="Rafas-MacBook-Pro.local",node_type="orch",round="3",tx_hash="0x882db1d6cd0609c1253aeed3d0ca8d99724e014f203b1dd140936e8bb2b1f626"} 1
```

So, the metric number will never be higher than `1`. That makes me think that we should use logging instead of metrics.